aws/aws.go

```diff
@@ -1,8 +1,9 @@
 package aws
 
 type AWS struct {
-	KMSKeyArn string `json:"kmsKeyArn" yaml:"kmsKeyArn"`
-	Masters   []Node `json:"masters" yaml:"masters"`
-	Region    string `json:"region" yaml:"region"`
-	Workers   []Node `json:"workers" yaml:"workers"`
+	KMSKeyArn  string   `json:"kmsKeyArn" yaml:"kmsKeyArn"`
+	Masters    []Node   `json:"masters" yaml:"masters"`
+	Region     string   `json:"region" yaml:"region"`
+	Workers    []Node   `json:"workers" yaml:"workers"`
+	PublicKeys []string `json:"publicKeys" yaml:"publicKeys"`
```

Comment:
Propagating the SSH keys for instance access via command line arguments would be better because this aligns with the other keys we propagate. 

I do not understand this. 

> Currently, the only way to inject keys is to modify the contents of cloudconfig (in k8scloudconfig repo), generate the bindata and copy it into vendor of aws-operator. That's a huge problem for us.

Lets hang.